src/TraceEvent/TraceEvent.Tests/AutomatedAnalysis/AnalyzerExecutionTests.cs

```diff
@@ -79,7 +79,7 @@ protected override AnalyzerExecutionResult Execute(AnalyzerExecutionContext exec
             if (processContext.Process.DisplayID == PID && ProcessDescription.Equals(processContext.Process.Description))
             {
                 Assert.Null(Issue);
-                Issue = new AnalyzerIssue("Test Title", "Test Description", "http://test-url");
+                Issue = new AnalyzerIssue(Guid.NewGuid(), "Test Title", "Test Description", "http://test-url");
```

Comment:
Can you please create constant Guid and pass it in here for testing purposes?

Reply:
Done.